FrostyEditor/ViewModels/Windows/WelcomeWindowViewModel.cs

```diff
@@ -31,6 +33,52 @@ public ProfileConfig(string inKey)
             Key = inKey;
             Path = Config.Get("GamePath", string.Empty, ConfigScope.Game, Key);
             Name = ProfilesLibrary.GetDisplayName(Key) ?? Key;
+            Icon = BitmapAssetValueConverter.Instance.Convert($"avares://FrostyEditor/Assets/Profiles/Icons/{ProfilesLibrary.GetInternalName(inKey)}.png");
```

Comment:
the icon/banner should probably be in a folder which the user can add, since profiles should be added through the user

Reply:
user should can add banner manually?
can u see [this](https://github.com/shoushou1106/SuperCheatedFrostyToolsuite/tree/2.0.0_ThemeSystem)? I think it would be better to make the banner part of the theme